ios/Wrappers/ConversationWrapper.swift

```diff
@@ -23,7 +23,7 @@ struct ConversationWrapper {
 			"createdAt": UInt64(conversation.createdAt.timeIntervalSince1970 * 1000),
 			"context": context,
 			"peerAddress": conversation.peerAddress,
-			"version": conversation.version == .v1 ? "v1" : "v2",
+			"version": "DIRECT",
```

Comment:
Note to future self that this will need to be noted in the release notes as a breaking change.